cpp/src/io/orc/reader_impl.cu

```diff
@@ -611,11 +894,13 @@ table_with_metadata reader::impl::read(size_type skip_rows,
   // Return column names (must match order of returned columns)
   out_metadata.column_names.resize(_selected_columns.size());
   for (size_t i = 0; i < _selected_columns.size(); i++) {
-    out_metadata.column_names[i] = _metadata->get_column_name(_selected_columns[i]);
+    out_metadata.column_names[i] = _metadata->get_column_name(0, _selected_columns[i]);
   }
-  // Return user metadata
-  for (const auto &kv : _metadata->ff.metadata) {
-    out_metadata.user_data.insert({kv.name, kv.value});
+
+  // XXX: Review question. Should metadata from all input files be included here as I am doing or
+  // just a single input file? Return user metadata
```

Comment:
I really don't know, haven't seen this used. Current code looks correct.

Reply:
I am not sure if you can have multiple user_data